### PR TITLE
Sqlite: add course

### PR DESCRIPTION
**Pull request by CleanMachine1**
## What does this PR do?
Add resource(s) 

### Why is this valuable (or not)?

SQLite is currently undocumented from what I see 

### How do we know it's really free?

Its a no signup + tutorialspoint website

### For book lists, is it a book? For course lists, is it a course? etc.

course

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -255,6 +255,7 @@
 * [SQL Foundations](https://www.udemy.com/course/sql-essentials-for-beginners/) - Sagar Uppuluri (Udemy)
 * [SQL Tutorial](https://www.youtube.com/playlist?list=PL_RGaFnxSHWr_6xTfF2FrIw-NAOo3iWMy)
 * [SQL Tutorial - Full Database Course for Beginners](https://www.youtube.com/watch?v=HXV3zeQKqGY) - Mike Dane (freeCodeCamp)
+* [SQLite Tutorial](https://www.tutorialspoint.com/sqlite/index.htm)
```
Comment (LuigiImVector) [inline courses/free-courses-en.md:258]:
```suggestion
* [SQLite Tutorial](https://www.tutorialspoint.com/sqlite/)
```

The right category is `books/free-programming-books-langs.md`.

Comment (CleanMachine1) [reply courses/free-courses-en.md:258]: It isn't a book.

Comment (LuigiImVector) [reply courses/free-courses-en.md]: It's a text book.

From [CONTRIBUTING.md](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md):
>Books vs. Courses
>
>Sometimes these can be hard to distinguish!
>
>Courses often have associated textbooks, which we would list in our books lists. Courses have lectures, exercises, tests, notes or other didactic aids. A single lecture or video by itself is not a course. A powerpoint is not a course.

There are tutorialspoint link both in the courses section and in the books section so maybe it's better to wait for indications from some other more experienced staffer.

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -2167,6 +2167,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 
 * [Developing Time-Oriented Database Applications in SQL](https://www2.cs.arizona.edu/~rts/tdbbook.pdf) - Richard T. Snodgrass (PDF)
 * [Introduction to SQL](https://github.com/bobbyiliev/introduction-to-sql) - Bobby Iliev (Markdown, PDF)
+* [SQLite Tutorial](https://www.tutorialspoint.com/sqlite/index.htm)
```
Comment (CleanMachine1) [inline books/free-programming-books-langs.md:2170]:
```suggestion
* [SQLite Tutorial](https://www.tutorialspoint.com/sqlite)
```

File: courses/free-courses-en.md (hunk 1 above)
Comment (davorpa) [reply courses/free-courses-en.md]: Maybe the past opened discussion #4573 could help, it talks about `TutorialsPoint`

File: books/free-programming-books-langs.md [hunk 3]
```diff
@@ -2167,6 +2167,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 
 * [Developing Time-Oriented Database Applications in SQL](https://www2.cs.arizona.edu/~rts/tdbbook.pdf) - Richard T. Snodgrass (PDF)
 * [Introduction to SQL](https://github.com/bobbyiliev/introduction-to-sql) - Bobby Iliev (Markdown, PDF)
+* [SQLite Tutorial](https://www.tutorialspoint.com/sqlite)
```
Comment (davorpa) [inline books/free-programming-books-langs.md:2170]: It's rare that linter warns about L1209 but maybe referes to place it below `SQL Queries Succinctly` (at L1173)

File: books/free-programming-books-langs.md [hunk 4]
```diff
@@ -2170,6 +2170,8 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [SQL For Web Nerds](http://philip.greenspun.com/sql/)
 * [SQL Notes for Professionals](http://goalkicker.com/SQLBook/) - Compiled from StackOverflow Documentation (PDF)
 * [SQL Queries Succinctly](https://www.syncfusion.com/ebooks/sql_queries_succinctly) - Nick Harrison
+* [SQLite Tutorial](https://www.tutorialspoint.com/sqlite)
+* ```
```
Comment (CleanMachine1) [inline books/free-programming-books-langs.md:2174]:
```suggestion
```

Comment (CleanMachine1) [reply books/free-programming-books-langs.md:2174]: I use the suggestions for quick editing of these long files and I think I pressed the suggestion button twice without cleaning/removing the extra backticks!

**Pull request merged**